docs/content/docs/reference/telemetry.md

```diff
@@ -89,27 +90,29 @@ A device telemetry report is formatted as JSON and currently looks like:
 The type of information contained within a project telemetry report is currently:
 
 - The last date you used `keystone dev` for this project, and
-- The resolved versions of any `@keystone-6` packages used by this project, and
+- The resolved package versions of any `@keystone-6` packages used by this project, and
+- The database type used by the project,
 - The number of lists for this project, and
 - The name and number of field types that you are using
 
 A project telemetry report is formatted as JSON and currently looks like:
 
 ```json
 {
-  "previous": "2022-11-23",
-  "versions": {
-    "@keystone-6/auth": "5.0.1",
-    "@keystone-6/core": "3.1.2",
+  "lastSentDate": "2024-11-23",
+  "packages": {
+    "@keystone-6/auth": "8.0.1",
+    "@keystone-6/core": "6.1.0",
     "@keystone-6/document-renderer": "1.1.2",
     "@keystone-6/fields-document": "5.0.2"
   },
+  "database": "postgresql",
```

Comment:
@DavidMulder0 as in https://github.com/keystonejs/keystone/pull/9290, with this missed in our telemetry policy - I will set this field to `NULL` for any remaining telemetry data that is to be aggregated.


```sql
UPDATE ProjectEvent SET database = NULL;
```

Reply:
We did disclaim this could happen in *What information is collected?*, but, this was missed on the _first_ pull request.